### DISCUSSION TITLE
Apt Pinning Priorities For Backport/Stable Packages

### DISCUSSION THREAD

**Discussion by munawarcmd** [SpiralLinux/SpiralLinux-project#356] [General]
Hi Geckos, is there anyway out I can copy Spiral's apt pinning priorities to my Vanilla Debian desktop? I'm currently not confident enough to set right priorities of my own and rather have Gecko's priorities instead. Also I apologise for my poor searching skill in gits and bothering geckos on this sub. Your help is highly appreciated.

(Pasted from Reddit)

**Comment by geckolinux** [comment]
Hi there, I assume you're referring to SpiralLinux. In that case, yes, you can copy the `/etc/apt/sources.list` , `/etc/apt/sources.list.d/spirallinux.list` and `/etc/apt/preferences.d/99backports` files from SpiralLinux to your vanilla Debian installation.

**Reply by munawarcmd** [reply]
Excellent! Also would it be possible for you to attach 3 of those files in a .txt file here? This becomes easier for me to download and paste in their respected directories. Kind regards.
